docs/guide/custom-ui.md

```diff
@@ -200,11 +200,11 @@ Before you override an editor modal, consider [adding a custom schema field type
 
 ## Adding custom context menu items
 
-We can add custom context menu items (edit mode) from within any module, targeting any Vue component that implements `AposModal`. The menu registration should happen in the initialization phase.
+Apostrophe offers a context menu that can be used to carry out certain operations on a document, such as 'preview', 'duplicate', and so on. We can add custom context menu items (edit mode) from within any module, targeting any Vue component that implements `AposModal`. The menu registration should happen in the initialization phase.
 
-Here is an example of how to add custom context menu item labeled "My Menu".
+Here is an example of how to add custom context menu item labeled "My Menu Item".
 
-![A custom context menu 'My Menu' in the Piece Editor Modal](/images/ui-custom-context-menu.png)
+![A custom context menu 'My Menu Item' in the Piece Editor Modal](/images/ui-custom-context-menu.png)
```

Comment:
"A custom context menu item," not "a custom context menu". The menu is the entire list.